cli/lsp/testing/lsp_custom.rs

```diff
@@ -92,8 +93,9 @@ pub enum TestRunKind {
 pub struct TestRunRequestParams {
   pub id: u32,
   pub kind: TestRunKind,
-  #[serde(skip_serializing_if = "Option::is_none")]
-  pub exclude: Option<Vec<TestIdentifier>>,
+  #[serde(skip_serializing_if = "Vec::is_empty")]
+  #[serde(default)]
+  pub exclude: Vec<TestIdentifier>,
```

Comment:
Changing things like this that come from the client changes the API with the client.  This message is specifically sent from the client, and you are now making an optional property mandatory. 😢   Why make a breaking change with the API?

Reply:
See https://serde.rs/field-attrs.html#default